include/librealsense2/h/rs_device.h

```diff
@@ -146,6 +146,53 @@ void rs2_connect_tm2_controller(const rs2_device* device, const unsigned char* m
 */
 void rs2_disconnect_tm2_controller(const rs2_device* device, int id, rs2_error** error);
 
+/**
+* Set tm2 camera intrinsics
+* \param[in] device       tm2 device
+* \param[in] sensor_id    specify fisheye lens
+* \param[in] intr         intrinsics value to be written to the device
+* \param[out] error       If non-null, receives any error that occurs during this call, otherwise, errors are ignored
+* \return 0 if the operation is completed successfully.
+*/
+void rs2_set_tm2_intrinsics(const rs2_device* device, int sensor_id, const rs2_intrinsics* intr, rs2_error** error);
```

Comment:
The functions shouldn't include "tm2" as there is nothing tm2-specific in the signature. 
Additionally, these functions shall later be reused for other devices, so the APIs shall remain generic

Reply:
@ev-mp, I can't be sure these set functions signature are good for non-tm2. Anyway, I'll try to make them generic.

Reply:
Now all removed and move to use rs2_sensor* and rs2_stream_profile*, more aligned with the existing get interfaces